symengine/cwrapper.cpp

```diff
@@ -170,34 +189,46 @@ TypeID basic_get_type(const basic s)
     return static_cast<TypeID>(s->m->get_type_code());
 }
 
-void symbol_set(basic s, const char *c)
+int symbol_set(basic s, const char *c)
```

Comment:
That's a really good idea. I was thinking of it, because now it's hard to distinguish between the instances of Exception Handling, and instances where an int is returned. I'll change that in the next commit.